definitions/json/EventToComplexTypes.json

```diff
@@ -3935,9 +3932,139 @@
     "CaseFieldID": "respondentCollection",
     "ListElementCode": "responseRespondentContactPreference",
     "EventElementLabel": "Contact preference (from the ET3 form)",
-    "FieldDisplayOrder": 36,
+    "FieldDisplayOrder": 37,
     "DisplayContext": "OPTIONAL"
   },
+  {
+    "ID": "Respondent",
+    "CaseFieldID": "respondentCollection",
+    "CaseEventID": "amendRespondentDetails",
+    "ListElementCode": "et3ResponseRespondentContactName",
+    "EventElementLabel": "Name of contact at respondent's address if not you as the representative",
+    "FieldDisplayOrder": 12,
+    "DisplayContext": "OPTIONAL",
+    "RetainHiddenValue": "No"
+  },
```

Comment:
Seems duplicate of entry with "FieldDisplayOrder": 38 if I am not reading it wrong.

Reply:
Ah yes, well spotted